### ISSUE TITLE
[Feature]: Platform Operator can follow manual instructions to configure ingress

### ISSUE THREAD

**Issue by acosta11** [cloudfoundry/cf-k8s-controllers#59]
### Blockers/Dependencies

Depends on #56. Extends the requirements to also include configuration of a load balancer in front of a cluster and the initial manual configuration of CFDomain resources in the absence of a v3/domains endpoint and corresponding controller reconciliation loop.

### Background

As a Platform Operator
I want manual instructions to configure my ingress and CF domains
So that I can receive external traffic to my platform

### Acceptance Criteria

GIVEN I am targeting a K8s cluster with the cf shim and contour installed
WHEN I follow manual instructions for the load balancer and CF domain configuration 
THEN I can externally access my cluster and see traffic from the load balancer routed on a real domain

### Dev Notes

* Assumes CF domain management is deferred
* Assumes consumption of the gateway API for ingress configuration is deferred
* We can start with instructions for GKE.
